### DISCUSSION TITLE
有没有评论区按回车直接发送的，每次发送总是忘记按ctrl

### DISCUSSION THREAD

**Discussion by ddqlsep** [the1812/Bilibili-Evolved#4838] [功能建议 / Ideas]
每次评论完按enter就换行，然后才删掉换行，再按ctrl+enter

**Comment by Geeyun-JY3** [comment]
`快捷键扩展`里无法设置吗？

**Comment by the1812** [comment]
用快捷键扩展可以改成 enter, 不过由于最近的 [Shadow DOM 评论区](https://github.com/the1812/Bilibili-Evolved/discussions/4804)问题, 可能还是不能生效的
